### PR TITLE
CodecoolerDAO test

### PR DESCRIPTION
**Pull request by idzia**
Please check my test to methods:
testGetCodecooler()
testCodecoolerId()
testCodecoolerIdByNotExistUser()

I have to change InventoryDAO.

### CODE REVIEW

File: src/main/java/Controller/CodecoolerController.java [hunk 1]
```diff
@@ -174,10 +174,10 @@ private String getCodecoolerInventory(Codecooler codecooler) throws SQLException
         JtwigTemplate template = JtwigTemplate.classpathTemplate("templates/codecoolerWallet.twig");
         JtwigModel model = JtwigModel.newModel();
 
+        InventoryDAO inventoryDAO = new InventoryDAO();
+        List<Integer> artifactId = inventoryDAO.getArtifactsOfCodecooler(codecoolerDao.getCodecoolerId(codecooler.getUserId()));
 
-        List<Integer> artifactId = InventoryDAO.getArtifactsOfCodecooler(codecoolerDao.getCodecoolerId(codecooler.getUserId()));
-
-        List<Artifact> inventory = InventoryDAO.getListOfArtifact(artifactId);
+        List<Artifact> inventory = inventoryDAO.getListOfArtifact(artifactId);
```
Comment (ziutekk) [inline src/main/java/Controller/CodecoolerController.java]: I am not really sure why you changed static to object call

File: src/main/java/DAO/FactoryDAO.java [hunk 2]
```diff
@@ -27,7 +27,7 @@ public QuestDAO getQuestDAO() {
     }
 
     public CodecoolerDAO getCodecoolerDAO() {
-        return new CodecoolerDAO(createConnection());
+        return new CodecoolerDAO(createConnection(), new InventoryDAO());
```
Comment (ziutekk) [inline src/main/java/DAO/FactoryDAO.java]: you should use factory to do this

Comment (idzia) [reply src/main/java/DAO/FactoryDAO.java]: is than

Comment (idzia) [reply src/main/java/DAO/FactoryDAO.java]: Create method getInventoryDAO() in FactoryDAO

**Pull request merged**